app/common/src/cljc/teet/user/user_roles.cljc

```diff
@@ -0,0 +1,24 @@
+(ns teet.user.user-roles)
+
+
+(defn has-role?
+  "Returns true if user has the given role.
+  If the input is a single role keyword, checks that the user has that role.
+  If the input is a collection of role keywords, checks that user has at least
+  one of the roles."
+  ([{roles :user/roles :as user} role-or-roles]
+   (cond
+     (nil? role-or-roles)
+     true
+
+     (keyword? role-or-roles)
+     (contains? roles role-or-roles)
+
+     :else
+     (some #(has-role? user %) role-or-roles))))
```

Comment:
Ok, so here the sematics are "needs to have at least one of the listed roles". Maybe update the docstring "Returns true if user has the *single* given role *or one of the multiple given roles*"